tests/rustdoc/alias-reexport.rs

```diff
@@ -0,0 +1,18 @@
+// aux-build:alias-reexport.rs
+// aux-build:alias-reexport2.rs
+
+#![crate_name = "foo"]
+#![feature(type_alias_type)]
+
+extern crate alias_reexport2;
+
+// @has 'foo/reexport/fn.foo.html'
+// FIXME: should be 'pub fn foo() -> Reexport'
+// @has - '//*[@class="rust item-decl"]' 'pub fn foo() -> u8'
```

Comment:
Seems like the feature is not complete yet. ;)

Well if there are improvements for it, we'll be able to see it right away.

Reply:
Are we calling normalize somewhere in rustdoc? That would turn `Reexport` back into `u8`. Isn't there a flag in rustdoc to not normalize in signatures?

Reply:
I'll check that tomorrow.

Reply:
It's this code:

https://github.com/rust-lang/rust/blob/c55d1ee8d4e3162187214692229a63c2cc5e0f31/src/librustdoc/clean/mod.rs#L2055-L2058